### PR TITLE
Bump to latest gvisor-tap-vsock

### PR DESCRIPTION
**Pull request by guillaumerose**

### CODE REVIEW

**Comment by guillaumerose** [comment]
/unhold

Bundle is pushed under 4.8.11. Let's wait for the CI.

**Comment by praveenkumar** [comment]
proxy test failure on this PR is expected.

**Comment by anjannath** [comment]
Tried on windows, works as expected. [start debug logs](https://gist.github.com/anjannath/438e43e6f42434809c0e90cab0f03a3e)

**Comment by anjannath** [comment]
Tried on windows, works as expected. [start debug logs](https://gist.github.com/anjannath/438e43e6f42434809c0e90cab0f03a3e)

**Comment by guillaumerose** [comment]
Great! Thanks @anjannath

File: cmd/crc/cmd/daemon.go [hunk 1]
```diff
@@ -65,10 +66,13 @@ var daemonCmd = &cobra.Command{
 		virtualNetworkConfig := types.Configuration{
 			Debug:             false, // never log packets
 			CaptureFile:       os.Getenv("CRC_DAEMON_PCAP_FILE"),
-			MTU:               4000, // Large packets slightly improve the performance. Less small packets.
+			MTU:               mtu(), // Large packets slightly improve the performance. Less small packets.
```
Comment (praveenkumar) [inline cmd/crc/cmd/daemon.go:69]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**